doc/changelog/07-commands-and-options/13345-master+doc-add-ml-path-not-exported.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  Clarify that :cmd:`Add ML Path` is not exported to compiled files
```

Comment:
```suggestion
  Clarify (in the documentation) that :cmd:`Add ML Path` is not exported to compiled files
```

Reply:
I would even remove the parentheses, ok?